selfdrive/car/toyota/interface.py

```diff
@@ -258,7 +258,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.19], [0.02]]
       ret.lateralTuning.pid.kf = 0.00007818594
 
-    elif candidate == CAR.LEXUS_IS:
+    elif candidate in [CAR.LEXUS_IS, CAR.LEXUS_ISH]:
       stop_and_go = False
       ret.safetyParam = 77
```

Comment:
@nelsonjchen perhaps push it with 100 or 130, I personally ran 130 and it can take some serious turns.
https://youtu.be/vxq5apbCoq4

Reply:
I am not familiar with tuning and this value doesn't go that high for any other Toyota. If @bisra2 is interested, maybe he can run a fork with that but I'm going to leave it as-is for this PR.